Core/Security/X509Certificate.py

```diff
@@ -32,28 +64,31 @@ def loadFromFile( self, certLocation ):
     Return : S_OK / S_ERROR
     """
     try:
-      fd = file( certLocation )
-      pemData = fd.read()
-      fd.close()
+      with file( certLocation ) as fd:
```

Comment:
I think using 
```
with open( certLocation, 'r') as fd:
```
is more understandable

Reply:
Done